src/Chess.hs

```diff
@@ -0,0 +1,11 @@
+module Chess (UnitType, Position, Side, Unit, Board, move) where
+
+
+data UnitType = Pawn | Rook | Horse | Squire | Queen | King
```

Comment:
There are 6 types of pieces which are king, queen, rook, bishop, knight, and pawn. Modify this or explain your thoughts with a comment.